### ISSUE TITLE
Backup e recuperação

### ISSUE THREAD

**Issue by DaianeAlvesRJ** [LibreCodeCoop/gestao-infraestrutura#15]
## Questões a responder
- Sendo a infraestrutura definida como código, é preciso fazer backup do código fonte dos sistemas?
- Qual será a ferramenta de backup utilizada? Justificar porque uma e não outra trazendo pontos positivos e negativos em uma tabela comparativa. Sugestões:
  - Duplicati
  - Bacula

## Rotina de backup
Trazer definição do que é.

Definir se terá uma rotina de backup global para o servidor ou se terá uma rotina de backup específica para cada sistema.

Definir quais dados serão inclusos em uma rotina de backup para cada sistema.

Cada conjunto deste gera uma rotina de backup?

## Política de retenção

Lista de possíveis políticas de retenção de dados.

Sugestão de qual é a política de retenção de dado padrão adotada para cada rotina de backup.

## Entregáveis
- [ ] Configurar política de retenção na ferramenta de infra como código
- [ ] Configurar o que fará backup para cada rotina de backup
- [ ] Instruções de como restaurar backup total
- [ ] Instruções de como restaurar backup de um único arquivo em uma única data
- [ ] Rotina de teste para garantir que o backup foi feito com segurança

**Comment by vitormattos** [comment]
Esta tarefa avançou algo? Está com  status de bloqueada.

**Comment by henmohr** [comment]
Sim, aqui: https://github.com/LibreCodeCoop/nextcloud-ha/tree/card-%2315
